draft-ietf-rtcweb-jsep.xml

```diff
@@ -2581,6 +2586,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         <t>The "a=rtcp-mux" line MUST only be added if
                         present in the most recent answer.</t>
 
+                        <t>An "a=rtcp-mux-only" line, as specified in <xref
+                        target="I-D.ietf-mmusic-mux-exclusive" /> Section 4, if
+                        and only if it was present in the most recent answer
+                        and the RTP/RTCP multiplexing policy is "require".</t>
```

Comment:
I think line 2592 is probably superfluous here; the only way one would have this in the most recent answer is if this condition were already true.